dtfs-central-api/test-helpers/test-data/eligibility-criteria-amendments.ts

```diff
@@ -0,0 +1,17 @@
+import { ObjectId } from 'mongodb';
+import { AmendmentsEligibilityCriteria, DEAL_TYPE, FACILITY_TYPE } from '@ukef/dtfs2-common';
+import { generateMockPortalUserAuditDatabaseRecord } from '@ukef/dtfs2-common/change-stream/test-helpers';
+
+export const anAmendmentsEligibilityCriteria = (): AmendmentsEligibilityCriteria => ({
```

Comment:
```suggestion
export const amendmentsEligibilityCriteria = (): AmendmentsEligibilityCriteria => ({
```

We don't really use indefinite articiles as a function prefix.

Reply:
DocString please.

Reply:
@abhi-markan we have historically for test helpers used `a` as a prefix e.g.: